test/fixtures.js

```diff
@@ -17,6 +17,8 @@ module.exports = nock('https://openshift.redhat.com')
 .reply(200, datareply, headers)
 .get('/broker/rest/cartridges', '*')
 .reply(200, datareply, headers)
+.get('/broker/rest/application/1a')
+.reply(200, datareply, headers)
```

Comment:
seem ok, but hard to maintain and debug.
alternative is to mock expected responses as required in test functions

(forgive mockito/jasmine combined syntax)
e.g. 
```
when(request).get(<dont care>).then(function() {
  return cb(null, mockRes);
});
<test code here>
```

not sure what would provide something like this